src/dispatch.js

```diff
@@ -11,6 +11,42 @@ import {
   ifElse,
 } from 'ramda';
 
+/**
+ * Can be used as a way to compose multiple invokers together to form polymorphic functions,
+ * or functions that exhibit different behaviors based on their argument(s).
+ * Consumes dispatching functions and keep trying to invoke each in turn, until a non-nil value is returned.
+ *
+ * Accepts a list of dispatching functions and returns a new function.
+ * When invoked, this new function is applied to some arguments,
```

Comment:
When invoked, each dispatching function is applied to all arguments supplied to this new function in sequence, until one of the dispatching functions returns a non-nil value.

Reply:
I stole this from converge description http://ramdajs.com/docs/#converge

Reply:
Fair enough. I just find the phrase 'some arguments' to be very vague.